### DISCUSSION TITLE
Tailwind Typography does little to nothing with markdown from strapi

### DISCUSSION THREAD

**Discussion by lmlTimolml** [tailwindlabs/tailwindcss#10668] [Help]
I am passing a richtext field from strapi to nextjs and have added tailwind typography as stated in every doc and "how-to"-video/article, the text looks like this in strapi:

_Lorem Ipsum is_ 
##simply dummy text the <u>industry</u>'s standard and **scrambled** 

- Listitems
- Listitems
- Listitems
- Listitems

The result after adding the simple className="prose":
_Lorem Ipsum is_ ##simply dummy text the <u>industry</u>'s standard and **scrambled** - Listitems - Listitems - Listitems - Listitems

What am I missing?

**Comment by wongjn** [comment]
It could be that the markdown is not being properly rendered into HTML. To be clear, the `@tailwindcss/typography` does not do this for you.
